src/main/java/org/launhcode/healthynutrition/controllers/CategoryController.java

```diff
@@ -0,0 +1,40 @@
+package org.launhcode.healthynutrition.controllers;
+
+import org.launhcode.healthynutrition.data.RecipeCategoryRepository;
+import org.launhcode.healthynutrition.models.RecipeCategory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@Controller
+@RequestMapping("recipeCategory")
+public class CategoryController {
+
+    @Autowired
+    private RecipeCategoryRepository recipeCategoryRepository;
+
+    @GetMapping("add")
+    public String displayCategoryForm(Model model){
+        model.addAttribute("title", "Add Category");
+        model.addAttribute(new RecipeCategory());
+        return "recipeCategory/add";
+    }
+
```

Comment:
minor: there should only be one space between these methods

Reply:
Sure, I will take care